gigamon/assets/logs/gigamon.yaml

```diff
@@ -0,0 +1,98 @@
+id: gigamon
+metric_id: gigamon
+backend_only: true
+
+facets:
+  - groups:
+      - Web Access
+    name: URL Path
+    path: http.url_details.path
+    source: log
+  - groups:
+      - Web Access
+    name: Status Code
+    path: http.status_code
+    source: log
+  - groups:
+      - DNS
+    name: DNS ID
+    path: dns.id
+    source: log
+  - groups:
+      - DNS
+    name: Response Code
+    path: dns.flags.rcode
+    source: log
+  - facetType: list
+    groups:
+      - gigamon
+    name: Application Name
+    path: app_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: dns_host
+    path: dns_host
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: dns_name
+    path: dns_name
+    source: log
+    type: string
+pipeline:
+  type: pipeline
+  name: gigamon
+  enabled: false
+  filter:
+    query: "source:gigamon"
+  processors:
+    - type: attribute-remapper
+      name: Map http_host` to `http.url_details.path`
+      enabled: true
+      sources:
+        - http_host
+      sourceType: attribute
+      target: http.url_details.path
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `http_code` to `http.status_code`
+      enabled: true
+      sources:
+        - http_code
+      sourceType: attribute
+      target: http.status_code
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `dns_transaction_id` to `dns.id`
+      enabled: true
+      sources:
+        - dns_transaction_id
+      sourceType: attribute
+      target: dns.id
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `dns_flags` to `dns.flags.rcode`
+      enabled: true
+      sources:
+        - dns_flags
+      sourceType: attribute
+      target: dns.flags.rcode
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: date-remapper
+      name: Mapping timestamp to a dateremapper
+      enabled: true
+      sources:
+        - ts
```

Comment:
The `ts` value in the log samples have values like: `Tue May 21 18:13:52 2024`. This is not a supported date format, more details in the doc [here](https://docs.datadoghq.com/service_management/events/pipelines_and_processors/date_remapper/). 

I see three attributes in the logs that might be timestamps: `http_request_ts`, `http_response_ts`, and the `ts` that you were trying to map. If you want to use the `ts`, then you'd have to create a grok parser that extracts the dates into a format that we support. If you want to use any of the. other request/response timestamps, then you'd need to convert them into an epoch time in ms (I'm not sure what the value corresponds to right now)